templates/compute-domain-daemon.tmpl.yaml

```diff
@@ -36,6 +36,8 @@ spec:
             touch /etc/nvidia-imex-null
             tail -f /dev/null & wait
           fi
+          # Emit nodes config for facilitating debug.
+          echo -e "/etc/nvidia-imex/nodes_config.cfg:\n\n" && cat /etc/nvidia-imex/nodes_config.cfg
```

Comment:
@jgehrcke i feel this alone is not enough to debug the connectivity among nodes. We should also run `nvidia-imex-ctl -a -q` this actully queries the connection. we should probably run this once after we start the daemon.